### ISSUE TITLE
Variable naming problems

### ISSUE THREAD

**Issue by RichardMuri** [arhik/SRT#11]
In the file Plot_Trend_Tests.R starting in line 25 there is what appears to be a global variable named q. This is not a descriptive name; either the implementation or the name should change such that someone who didn't write the code can tell what it does.

**Comment by arhik** [comment]
You are right. Its allens code lets make Allen work on it. If you can clean it go ahead. Check for similar functionality implemented by me within server.R. If you cant find it let me know. I will point you to that portion if time permits so that you can make necessary changes.
